Chutzpah/Chutzpah.csproj

```diff
@@ -40,6 +40,12 @@
     <Reference Include="AntiXssLibrary">
       <HintPath>..\packages\AntiXSS.4.2.1\lib\net40\AntiXssLibrary.dll</HintPath>
     </Reference>
+    <Reference Include="EnvDTE, Version=8.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">

```

Comment:
Having Chutzpah core library take a reference on envDTE seems bad. This means chutzpah.exe will have a dependency on a VS install.